rest-server/build.sbt

```diff
@@ -164,6 +162,7 @@ lazy val admin = (project in file("admin")).dependsOn(core).
 
 lazy val server = (project in file("server")).dependsOn(core, common, engines, admin).settings(
   commonSettings,
+  generateBuildInfo,
```

Comment:
@pferrel in case of ^^^ usage, this should be moved to admin module. (I'll push)